CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ after connecting (without the previous need to subscribe to individual rooms).
 ### Changed
 
 - Upgraded OkHttp to the highest version supporting Android 4.4.
```

Comment:
maybe let's change to Updated here too, sorry for not using consistent terms with the recent releases

Reply:
Updated.!